test/functional/apps/dashboard_elements/controls/options_list/options_list_dashboard_interaction.ts

```diff
@@ -110,7 +118,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             operation: 'is one of',
             value: ['bark', 'bow ow ow', 'ruff'],
           });
-          await dashboard.waitForRenderComplete();
```

Comment:
The controls don't use the `RenderCompleteDispatcher` **at all** so, unless we are checking something about the **charts** (which do use this), there isn no need to call `waitForRenderComplete`